apps/sasquatch/src/main/res/values/env.xml

```diff
@@ -5,7 +5,8 @@
 
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyDashes">
     <string name="app_secret" tools:ignore="MissingTranslation">45d1d9f6-2492-4e68-bd44-7190351eb5f3</string>
-    <string name="log_url" tools:ignore="MissingTranslation" />
+    <string name="log_url" tools:ignore="MissingTranslation">https://in.appcenter.ms</string>
```

Comment:
We should not do that otherwise we never test default SDK value. We can read the default url from code, revert that.